python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2773,6 +2774,12 @@ def from_tensorflow(self, graph, layout="NHWC", shape=None, outputs=None):
             if freezed_ops:
                 raise Exception("Graph is not frozen. Provide a frozen graph. "
                                 "Found operators {}".format(freezed_ops))
+            stateful_ops = [op for op in missing_operators
+                            if op in self._main_graph_proto._stateful_ops_list]
+            if stateful_ops:
+                raise Exception("Found stateful operators in this graph {}. " \
```

Comment:
Again this will display exception with only stateful missing ops (Doesn't show normal missing ops.).
I don't think we need separate list for stateful missing ops. Just add them to missing_operators list and display as one.

Reply:
Okay. merged into missing op list itself and removed seperate exception code for stateful ops.